### DISCUSSION TITLE
OG Image is not showing on twitter, despite the tags being correctly set and viewable on other platforms.

### DISCUSSION THREAD

**Discussion by ybhrdwj** [vercel/community#6386] [Help]
### Summary

None of my vercel projects (https://www.pondy.life/) and aajdrydayhaikya.com's OG images are showing up on twitter:
- they once used to, but suddenly stopped. 
- I can't access the Opengraph section in my deployments it says "error loading dashboard" 

What am I doing wrong?
![CleanShot 2024-04-04 at 03 01 01@2x](https://github.com/vercel/community/assets/22580465/f67914e6-adaf-4c2c-9184-27602f2f0d38)
![CleanShot 2024-04-04 at 03 01 30@2x](https://github.com/vercel/community/assets/22580465/d87f5ecd-d4d7-47ea-a9b2-94cb0c980dd5)
![CleanShot 2024-04-04 at 03 02 00@2x](https://github.com/vercel/community/assets/22580465/623fe50d-7c63-407c-867d-6149e70fc79c)

- attached: code snapshot
- error snapshot

### Example

pondy.life

### Steps to Reproduce

1. paste pondy.life on twitter
2. see the opengraph image not load

**Comment by kirankulkarni** [comment]
I am also facing this problem. Have you figured out a solution?
